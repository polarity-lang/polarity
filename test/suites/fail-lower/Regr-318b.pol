data Foo(a: Type) {}

let foo: Foo(Type, Type) { ? }
