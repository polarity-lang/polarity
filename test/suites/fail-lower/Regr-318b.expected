L-014

  × Too many arguments provided
   ╭─[Regr-318b.pol:3:14]
 2 │ 
 3 │ let foo: Foo(Type, Type) { ? }
   ·              ────
   ╰────
